added icons bluos, calc_you, ghostery, helium_wallet and one_protect

je223xa opened the pull request:
Hey there, 
my first contribution ever to anything. But I love this project and think it's worth spending some time on. 
Feel free to come back to me with editions or changes, I'd love to contribute more if what I'm doing is good enough for your standards

alchemiker commented:
Hello @je223xa ,

Welcome to Arcticons and congratulations on your first pull request! The icons look good to me at first glance, but Donno, the maintainer of this project, will probably come back to you with more detailed feedback on them.

Besides uploading the SVGs, you also need to tell Arcticons what app your icon belongs to. For this, you need to edit the appfilter.xml file in your Fork of Arcticons in the folder "other" > appfilter.xml (or if you can't find it, a direct link to the file in your fork here:
https://github.com/je223xa/Arcticons/blob/main/other/appfilter.xml

The entries are ordered roughly alphabetically. You have to add for each of your icon an entry, that looks something like this:

`<!-- Ghostery -->`
`<item component="ComponentInfo{com.ghostery.android.ghostery/com.ghostery.android.ghostery.App}" drawable="ghostery"/>`

The word in quotation marks behind "drawable" should be the name of your SVG file.

How to find this piece of code, by which an app is internally called? There are several options:

1. The requests.txt file: at "other" > requests.txt ([here](https://github.com/Arcticons-Team/Arcticons/blob/main/other/requests.txt)) you can search the list of the apps requested by other users with Ctrl + F (this is where I found Ghostery for the example before)
2. The request function in the Arcticons app: if you have the app you made the icon for installed, you can use the "Icon Request" feature. It opens up an email draft which contains the code for the app you requested.
3. You can use the app [IconRequest](https://github.com/Kaiserdragon2/IconRequest) which displays this code for all apps which you have on your phone.

This is just a longer explanation of what is summarized in the [Contribution Guide](https://github.com/Arcticons-Team/Arcticons/blob/main/CONTRIBUTING.md) for this project. I myself had some difficulty to understand what is expected from me when I did my first commit to this project some months ago, that's why I wrote it in more detail for you. 

Please feel free to ask if something is still unclear, and thanks for contributing!

je223xa commented:
Hey there, thank you for providing the detailed information, should've read more carefully I suppose.
I have added the entries to `appfilter.xml` , I have not added an entry for OneProtect since it already had an entry so I've just updated that. Maybe you guys can decide wether to use mine, or the number_1 icon

Donnnno commented:
Hi! nice work on the icons! I just made some small changes:
- sizing: some icons were really small, while others were kinda too big.
- line-width: made sure every line is 1px 
- stroke color: I noticed not all them were #ffffff


<img width="744" alt="Scherm­afbeelding 2024-01-13 om 13 07 45" src="https://github.com/Arcticons-Team/Arcticons/assets/31142286/a11c6bf1-c87f-4241-9c20-cb55eddb56fb">
